packaging/smf/propolis-server/manifest.xml

```diff
@@ -15,16 +15,16 @@
     </method_environment>
   </method_context>
   <exec_method type='method' name='start'
-    exec='ctrun -l child -o noorphan,regent /opt/oxide/propolis-server/bin/propolis-server run /var/svc/manifest/site/propolis-server/config.toml %{config/server_addr} --metric-addr %{config/metric_addr} &amp;'
+    exec='/opt/oxide/lib/svc/manifest/propolis/propolis.sh'
     timeout_seconds='0' />
   <exec_method type='method' name='stop' exec=':kill' timeout_seconds='0' />
 
   <property_group name='config' type='application'>
-    <propval name='server_addr' type='astring' value='unknown' />
-  </property_group>
-
-  <property_group name='config' type='application'>
-    <propval name='metric_addr' type='astring' value='unknown' />
+    <propval name='datalink' type='astring' value='unknown' />
+    <propval name='gateway' type='astring' value='unknown' />
+    <propval name='listen_addr' type='astring' value='127.0.0.1' />
+    <propval name='listen_port' type='astring' value='17000' />
```

Comment:
Worth changing to the default `12400` propolis-server uses?

Reply:
Done